filebeat/prospector/prospector.go

```diff
@@ -31,18 +33,20 @@ type Prospector struct {
 	wg               sync.WaitGroup
 	channelWg        sync.WaitGroup // Separate waitgroup for channels as not stopped on completion
 	harvesterCounter uint64
+	Hash             uint64
```

Comment:
As the type has to be configured, it is also hashed and and part of the ID. To better understand what an ID stands for I prefer that we log the config of a prospector with its identifier instead. I think that brings more usability to the user.